compiler/rustc_traits/src/normalize_projection_ty.rs

```diff
@@ -36,7 +36,7 @@ fn normalize_projection_ty<'tcx>(
                 &mut obligations,
             );
             fulfill_cx.register_predicate_obligations(infcx, obligations);
-            Ok(NormalizationResult { normalized_ty: answer })
+            Ok(NormalizationResult { normalized_ty: answer.ty().unwrap() })
```

Comment:
seems okay, as all the users of `normalize_projection_ty` expect a type, but please leave a comment to that regard